MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -1725,6 +1723,13 @@ public boolean recruitPerson(Person p, PrisonerStatus prisonerStatus, boolean gm
             }
         }
 
+        if(log){
+            formerSurname = p.getSurname().equals(formerSurname) ? "" : String.format(" (Formerly %s) ", formerSurname);
+            String add = !prisonerStatus.isFree() ? (prisonerStatus.isBondsman() ? " as a bondsman" : " as a prisoner")
+                : "";
+            addReport(String.format("%s%s has been added to the personnel roster%s.", p.getHyperlinkedName(), formerSurname, add));
```

Comment:
I know this isn't your code, but can we get these in-code report strings moved to the campaign resource bundle?

Reply:
Yep, I think I did this the way you wanted.